identity_vc/src/lib.rs

```diff
@@ -1,7 +1,27 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        assert_eq!(2 + 2, 4);
-    }
+#[macro_use]
```

Comment:
I guess the change from `#[macro_use] extern crate` to `use` is quite consistent with other items so 👍 .

I think the `mod` change was a disaster 👎  I still don't see that structure often in practice and I think it makes the mental model worse. I thought it was largely added to eliminate the restriction of **requiring** `mod.rs` when using sub-modules.

Reply:
I actually was really glad when they decided to remove the mod files.  Though admittedly, I am not a huge fan of the Rust module system in general.  That you have to create a file when you want to use a folder whether or it be inside or outside of the folder is always going to feel a little awkward.  Still its a bit less confusing to have distinctly named files rather than 30 files called mod.rs especially if you have only that mod file in the folder.